internal/sourcemap/generator.go

```diff
@@ -333,13 +335,26 @@ func (gen *Generator) RawSourceMap() *RawSourceMap {
 	}
 }
 
-// Gets the string representation of the source map
-func (gen *Generator) String() string {
+func (gen *Generator) bytes() []byte {
 	buf, err := json.Marshal(gen.RawSourceMap())
 	if err != nil {
 		panic(err.Error())
 	}
-	return string(buf)
+	return buf
+}
+
+// Gets the string representation of the source map
+func (gen *Generator) String() string {
+	return string(gen.bytes())
+}
+
+func (gen *Generator) Base64DataURL() string {
+	const prefix = "data:application/json;base64,"
+	data := gen.bytes()
+	buf := make([]byte, len(prefix)+base64.StdEncoding.EncodedLen(len(data)))
```

Comment:
this can also be done via `strings.Builder` with `Grow` and `Write` if preferred, it's essentially the same `unsafe.String` trick under the hood